lib/diagnostic.js

```diff
@@ -29,8 +34,15 @@ const sum = function (/* previous, current */) {
 // in the array returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.some`
-const any = function (/* array, callback */) {
-
+const any = function (array, callback) {
+  for (let i = 0; i < array.length; i++) {
+    if (callback(array[i], i, array)) {
+      return true;
+    }
+    else {
+      return undefined;
+    }
+  }
 };
```

Comment:
Two issues here: 
1) Why is your `else` clause returning `undefined` instead of `false`?
2) Where is that 2nd return statement happening? Look at its position relative to your if-statement and your for-loop.